Formula/cromshell-alpha.rb

```diff
@@ -0,0 +1,55 @@
+class CromshellAlpha < Formula
+  include Language::Python::Virtualenv
+
+  desc "Python tool to interact with a Cromwell server"
+  homepage "https://github.com/broadinstitute/cromshell"
+  url "https://storage.googleapis.com/bshifaw-bucket/cromshell-alpha.2.0.0.tar.gz"
```

Comment:
Currently set to a google bucket until releases can be made for cromshell2.0 in the cromshell repo

Reply:
Oh, interesting... we can add tags for alpha if we want.